src/intervals/intervals.jl

```diff
@@ -5,37 +5,38 @@
 
 ## Interval type
 
+if haskey(ENV, "IA_VALID")
+    const validity_check = true
+else
+    const validity_check = false
+end
+
 abstract type AbstractInterval{T} <: Real end
```

Comment:
(This was not there when I began reviewing.)
My suggestion (see elsewhere) is that the default is to have the validity being checked.